temp2.js

```
@@ -0,0 +1,1097 @@
+                  >
+                    {msg.sender === "user" ? (
+                      <div
+                        style={{
+                          backgroundColor:
+                            storedTheme === "true" ? "#F9F9F9" : "#282C34",
+                          padding: 16,
+                          color: storedTheme === "true" ? "#01020A" : "#fff",
+                          margin: 4,
+                        }}
+                        className="text-black dark:text-white text-xs"
+                        dangerouslySetInnerHTML={{ __html: msg.text }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)